CI/bench/multimodal/custom_user_dir/multimodal_frameworks_template.yaml

```diff
@@ -4,4 +4,4 @@ AutoGluon_multimodal_best:
   params:  # MultimodalPredictor.fit(params) # can add the actual job time limit here
     presets: best_quality
     hyperparameters:
-      optimization.max_epochs: 10 # 10 is default
+      optim.max_epochs: 10 # 10 is default
```

Comment:
General question: minor bug fixes / updates have been made over the last several months in AutoMM mainline. Does this PR incorporate those changes as well (via rebasing during the period of development), or does the PR overwrite them?

If it rebased on them, were there any major merge conflicts resolved that are part of this PR worth mentioning?

Reply:
This PR is based on the most recent AutoMM mainline. I didn't notice other major conflicts except for the configuration changes.